README.md

```diff
@@ -1,64 +1,27 @@
-# OpenShift Installer
-
-## Supported Platforms
-
-* [AWS](docs/user/aws/README.md)
-* [AWS (UPI)](docs/user/aws/install_upi.md)
-* [Azure](docs/user/azure/README.md)
-* [Bare Metal (UPI)](docs/user/metal/install_upi.md)
-* [Bare Metal (IPI) (Experimental)](docs/user/metal/install_ipi.md)
-* [GCP](docs/user/gcp/README.md)
-* [GCP (UPI)](docs/user/gcp/install_upi.md)
-* [Libvirt with KVM](docs/dev/libvirt/README.md) (development only)
-* [OpenStack](docs/user/openstack/README.md)
-* [OpenStack (UPI) (Experimental)](docs/user/openstack/install_upi.md)
-* [oVirt](docs/user/ovirt/install_ipi.md)
-* [vSphere](docs/user/vsphere/README.md)
-* [vSphere (UPI)](docs/user/vsphere/install_upi.md)
-
-## Quick Start
-
-First, install all [build dependencies](docs/dev/dependencies.md).
-
-Clone this repository. Then build the `openshift-install` binary with:
-
-```sh
-hack/build.sh
+# High Availability Single Node Openshift Setup Operator
+
+Provide the ability for exactly two Single Node OpenShift clusters to operate as a **predefined** pair in an **active-active** configuration, detect when its peer has died and **automatically** take over its workloads after ensuring it is safe to do so.
+
+## Pre-requisites
+* 2 SNO Clusters.
+* Deployments that will be managed by HA Layer already exist.
+
+## Installation
+* Deploy This operator to each SNO cluster.
+* Load the yaml manifest of the HASNO (for each SNO cluster).
+
+## Example CRs
+An example HASNO object.
+```yaml
+   apiVersion: app.hasno.com/v1alpha1
+   kind: HALayerSet
+   metadata:
+     name: halayerset-sample
+   spec:
+     # Add fields here
+     node1address: "192.168.126.10"
+     node2address: "192.168.126.11"
+     nodename: "cluster1"
+     authkey: "brt0ANAw6miMbQU6SiH2gRFXEZJSFKLxtynt7CjhAqtN1GBFR3JR6xP3aZHWARjK"
```

Comment:
Seems to be out-of-date with the new structure

Reply:
Indeed, still on my todo list :+1: